libs/langchain/langchain/schema/runnable/config.py

```diff
@@ -97,8 +97,8 @@ def get_config_list(
     Helper method to get a list of configs from a single config or a list of
     configs, useful for subclasses overriding batch() or abatch().
     """
-    if length < 1:
-        raise ValueError(f"length must be >= 1, but got {length}")
+    if length < 0:
```

Comment:
I think additional code is needed in default batch implementation since it assumes that the list of configs cannot be empty

```
        """
        configs = get_config_list(config, len(inputs))

        def invoke(input: Input, config: RunnableConfig) -> Union[Output, Exception]:
            if return_exceptions:
                try:
                    return self.invoke(input, config, **kwargs)
                except Exception as e:
                    return e
            else:
                return self.invoke(input, config, **kwargs)

        # If there's only one input, don't bother with the executor
        if len(inputs) == 1:
            return cast(List[Output], [invoke(inputs[0], configs[0])])

        with get_executor_for_config(configs[0]) as executor:
            return cast(List[Output], list(executor.map(invoke, inputs, configs)))
```

Reply:
Good point! updated